Makefile

```diff
@@ -118,5 +120,18 @@ clean:
 		$(CURDIR)/bin \
 		$(CURDIR)/pkg
 
+# We support all the same CLI flags for connecting to Consul as the main CLI.
+# Rather than import the root Consul module, which is not supported, we copy the
+# relevant module. This is similar to what we do in consul-dataplane.
+# We don't include config_test.go because it pulls in a directory with test data.
+.PHONY: copy-http-flags
+copy-http-flags: ## copy http flags modules from consul
+	for file in config.go http.go http_test.go merge.go usage.go; do \
+		curl --fail https://raw.githubusercontent.com/hashicorp/consul/main/command/flags/$$file | \
+		sed 's/BUSL-1.1/MPL-2.0/' | \
+		sed '1s:^:// Code generated by make copy-http-flags. DO NOT EDIT.\n:' | \
```

Comment:
👍🏻 given the current options and level of investment in this repo, nice solution.